src/lib/ui/thermostat/FanModeMenu.tsx

```diff
@@ -1,11 +1,11 @@
-import { CheckBlackIcon } from 'lib/icons/CheckBlack.js'
 import { ChevronDownIcon } from 'lib/icons/ChevronDown.js'
 import { FanIcon } from 'lib/icons/Fan.js'
 import { FanOutlineIcon } from 'lib/icons/FanOutline.js'
 import { Menu } from 'lib/ui/Menu/Menu.js'
-import { MenuItem } from 'lib/ui/Menu/MenuItem.js'
+import { ThermoModeMenuOption } from 'lib/ui/thermostat/ThermoModeMenuOption.js'
 
-type Mode = 'auto' | 'on'
+const modes = ['auto', 'on'] as const
```

Comment:
The SDK should have a type for this one too.

Reply:
I don't see any references to this in `models.ts`

Reply:
https://github.com/seamapi/javascript/pull/278

Reply:
Cool; used in [0c0e45e](https://github.com/seamapi/react/pull/442/commits/0c0e45e9a3e084a3e0d49b5d72b24aec28e184ff)